### DISCUSSION TITLE
Control upper/lower casing of key words in scripts?

### DISCUSSION THREAD

**Discussion by mickeyjf** [dbeaver/dbeaver#13435] [Q&A] [answered]
I am mostly using postgres, which lower cases things by default. I assume that is why when I write a query in a SQL Editor it is automatically lower cased, and that it might not do that if I was using a DB other than Postgres (?). 

However, after testing my scripts I often want to move them into a file for later automatic execution, and I like to have the SQL key words upper cased so that they stand out:

"SELECT not_a_keyword FROM my_table WHERE..." kinda thing.

I see that there is a option to upper or lower case everything, but not exclusively the key words. Obviously, begin able to do this is no more than a mere convenience, but it would be nice.

**Comment by mickeyjf** [comment]
Or maybe there is already a setting to control this, and I'm just not finding it?

**Comment by uslss** [comment, marked as answer]
Keyword case could be set in Preferences
![image](https://user-images.githubusercontent.com/31996417/128149941-e1d1b216-d478-421a-92fc-0f91ae9957ab.png)
